libraries/ESP8266WebServer/src/ESP8266WebServer-impl.h

```diff
@@ -22,11 +22,14 @@
 
 
 #include <Arduino.h>
-#include <libb64/cencode.h>
+#include <base64.h>
 #include "WiFiServer.h"
 #include "WiFiClient.h"
 #include "ESP8266WebServer.h"
 #include "FS.h"
+
+using namespace esp8266webserver;
```

Comment:
Don't do this, at least not in a header file. This imports everything in esp8266webserver into the global namespace, which defeats the purpose of using a namespace. 
It is sometimes ok to do this in a cpp file, but even then it's better to import into the cpp's anonymous namespace and not into the global one.

Reply:
I am actually not able to spot something that is outside the namespace in "nm" symbol dump. can you elaborate an example? the using doesn't change namespace, it just avoid me having to put a quadrillion esp8266webserver:: prefixes to all the declarations.. 

I can do that if your concern is the "using" part.

Reply:
Although I'm not sure I understood the concern, because there are no globals only statics (which are not exported hence don't pollute and don't care in which namespace they're declared), I moved all #include's out of the -impl.h now and have my duplicate-import problem solved this way.